llama.cpp

```diff
@@ -12459,6 +12459,122 @@ int32_t llama_token_to_piece(const struct llama_model * model, llama_token token
     return 0;
 }
 
+int32_t llama_chat_apply_template_internal(std::string &dest, std::string chat_template, std::vector<const llama_chat_message *> conversation, bool add_ass);
```

Comment:
This declaration is not needed

Reply:
Because we have `-Wmissing-declarations`, the compiler throws a warning without this declaration.

Reply:
Make it `static` instead.

Reply:
Thanks! That does the trick